docs/changelog.rst

```diff
@@ -4,6 +4,18 @@ Changes in IPython kernel
 5.1
 ---
 
+5.1.3
+*****
+
+5.1.3 Includes several bugfixes and internal logic improvements.
+
+- Fix comm shutdown behavior (:ghpull: `433`, :ghpull: `435`)
```

Comment:
```suggestion
- Fix comm shutdown behavior by adding a ``deleting`` option to ``close`` which can be set to prevent registering new comm channels during shutdown (:ghpull: `433`, :ghpull: `435`)
```

Reply:
:+1:

Reply:
(can't commit the suggestion though)